src/sdl2/video.rs

```diff
@@ -373,7 +373,7 @@ impl DisplayMode {
 
     pub fn from_ll(raw: &ll::SDL_DisplayMode) -> DisplayMode {
         DisplayMode::new(
-            raw.format as u32,
+            pixels::PixelFormatEnum::from_u32(raw.format as u32).unwrap(),
```

Comment:
Panicking here is pretty cumbersome if the pixel format is not known by SDL, don't you think ? Say RGBA1010102 is added later in SDL (but not updated in rust-sdl2), from_u32 won't recognize this unknown format and will panic.

Try `PixelFormatEnum::from_u32(raw.format as u32).unwrap_or(PixelFormatEnum::Unknown)`, that way the PixelFormat will only be Unknown instead of crashing :)